pages/1.12/installing/evaluation/index.md

```diff
@@ -3,22 +3,22 @@ layout: layout.pug
 navigationTitle:  Evaluation
 title: Evaluation
 menuWeight: 10
-excerpt: Installing DC/OS for evaluation using Terraform templates
+excerpt: Installing DC/OS for evaluation using Universal Installer
 ---
 
 This page explains about Terraform and the evaluation methods that are supported for both DC/OS OSS (default) and DC/OS Enterprise installation. You can evaluate the installation of DC/OS cluster based on your requirements.
 
-# Overview of Terraform 
+# Overview of Universal Installer 
 A number of different installation methods have emerged to manage the life-cycle of DC/OS on a set of nodes in a cluster. These installation methods include AWS CloudFormation templates, Azure ARM templates, Ansible Playbooks, dcos-launch, dcos-gcp, and terraform-dcos. Each of these methods were designed to solve a particular use case, and therefore had some limitations around supporting the full life-cycle of (provision, deploy, install, upgrade, decommission) of DC/OS. For example, both AWS CloudFormation and Azure ARM template solutions do not support the upgrade process in DC/OS after the cluster is deployed the first time. Likewise, `dcos-launch` and `dcos-gcp` were only designed to run well in a test environment, and are not usable in a production setting. Only `terraform-dcos` has support for (most of) the full life-cycle in a production setting with the limitation that decommissioning is not fully supported.
 
-Terraform is an open source infrastructure automation tool which uses templates to manage infrastructure for multiple public cloud providers, service providers, and on-premises solutions. Terraform creates your infrastructure, configures resources, and manages communication between agents. The purpose of this tool is to automate most of the manual efforts of managing and maintaining distributed systems.
+Terraform is an open source infrastructure automation tool which uses templates to manage infrastructure for multiple public cloud providers, service providers, and on-premises solutions. Terraform creates your infrastructure, configures resources, and manages communication between agents. The purpose of this tool is to automate most of the manual efforts of managing and maintaining distributed systems. The Universal Installer is built on top of Terraform.
 
-The primary goals of using Terraform are as follows:
+The primary goals of using Universal Installers are as follows:
```

Comment:
The primary goals of using "The Universal Installer" (there is only one, which is our tool)